acme/setup.py

```diff
@@ -53,7 +53,7 @@
     version=version,
     description='ACME protocol implementation in Python',
     url='https://github.com/letsencrypt/letsencrypt',
-    author="Let's Encrypt Project",
+    author="Electronic Frontier Foundation",
```

Comment:
I'm fine with either "EFF and others" or "The Certbot project".  We could also make a Contributors file.  Github is almost good enough to make that unnecessary, but not quite &mdash; it misses people who don't have the right emails associated with their accounts.  For instance @schoen has made 681 commits to this repo but isn't in  [the contributor list](https://github.com/letsencrypt/letsencrypt/graphs/contributors).